examples/cryptocurrency/Cargo.toml

```diff
@@ -18,12 +18,19 @@ circle-ci = { repository = "exonum/exonum" }
 
 [dependencies]
 exonum = { version = "0.9.0", path = "../../exonum" }
+exonum-crypto = { path = "../../crypto" }
+exonum_derive = { path = "../../exonum_derive" }
```

Comment:
Please add `version` to path dependencies. It is required in order to use a crate from crates.io.

Here and in other Cargo.toml files.